src/mi2.ts

```diff
@@ -1052,5 +1049,52 @@ export class MI2 extends EventEmitter implements IDebugger {
         if(code<900000) code+900000;
         strCode = "" + code;
         return strCode;
-      }
+    }
+    
+    // Opens a terminal to show the application screen - gdbtty
+    createTerminal(gdbttytype, sleepVal, target){
+        if(!gdbttytype || gdbttytype === "xterm"){
+            let dispTarget = (target.length>50)?"..."+target.substr(target.length-50,target.length):target;
+            const xterm_args = [
+                "-title","GnuCobol Debug - "+dispTarget,
+                "-fa", "DejaVu Sans Mono", 
+                "-fs", "14",
+                "-e", "/usr/bin/tty;"+ 
+                "echo 'GNUCOBOL DEBUG';"+ 
+                "sleep "+sleepVal+";"
+                ]
+            
+            const xterm_process = ChildProcess.spawn("xterm", xterm_args, {
+                detached: true,
+                stdio: 'ignore',
+            });
+            xterm_process.unref();
+        }else{
+            let terminal = this.selectTerminal();
+            if (!terminal) {
+                terminal=vscode.window.createTerminal({
+                    name: `GnuCobol Debug Display #${NEXT_TERM_ID++}`, 
+                    location: vscode.window.activeTextEditor                
+                    } as any
+                );
+            }    
+            terminal.show();
+            terminal.sendText("trap '' 2;")
+            terminal.sendText("clear;sleep "+sleepVal+";");
+        }
+    }
+
+    // Find the terminal used in debug in vscode - gdbtty
+    selectTerminal(): vscode.Terminal | undefined {
+        const terminals = <vscode.Terminal[]>(<any>vscode.window).terminals;
+        let itemTerm: vscode.Terminal = undefined;
+        for (var property in terminals) {
+            if (terminals.hasOwnProperty(property)) {
+                if(terminals[property].name.includes("GnuCobol Debug Display")){
```

Comment:
please use the correct spelling: `GnuCOBOL`, here and above

Reply:
Adjusted.